File: src/main/kotlin/keno/ToyKotlin/lectures/lecture2/StringPrint.java
package keno.ToyKotlin.lectures.lecture2;

public class StringPrint {
    public static void main(String[] args) {
        String[] arr = {"apple", "orange", "candy"};
        int choice = 1;
        String yourChoice;
        if (choice < 0) {
            yourChoice = arr[0];
        } else {
            yourChoice = arr[2];
        }
        System.out.println("This is your choice: " + yourChoice);
    }
}

File: src/main/kotlin/keno/ToyKotlin/lectures/lecture2/StringPrint.kt
package keno.ToyKotlin.lectures.lecture2
fun main(args: Array<String>) {
    val arr = arrayOf("apple", "orange", "candy")
    val choice = 1
    var yourChoice = if (choice < 0) arr[0] else arr[2]
    println("This is your choice: $yourChoice")
}